src/components/checkDraw.js

```diff
@@ -0,0 +1,9 @@
+export function checkDraw(gameState, winningMessage) {
+  if (
+    gameState[0].indexOf(null) === -1 &&
+    gameState[1].indexOf(null) === -1 &&
+    gameState[2].indexOf(null) === -1
+  ) {
+    return (winningMessage.innerText = 'It`s a draw!');
```

Comment:
There is something very weird going on in this line. You're returning a value and assigning it at the same time

Reply:
fixed